pkg/pipeline/utils/sasl.go

```diff
@@ -0,0 +1,40 @@
+package utils
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/netobserv/flowlogs-pipeline/pkg/api"
+	"github.com/segmentio/kafka-go/sasl"
+	"github.com/segmentio/kafka-go/sasl/plain"
+	"github.com/segmentio/kafka-go/sasl/scram"
+)
+
+func SetupSASLMechanism(cfg *api.SASLConfig) (sasl.Mechanism, error) {
```

Comment:
nitpick: Since the function is tied to the kafka-go type `sasl.Mechanism` may be it should be in the kafka ingest file.